application/leds.c

```diff
@@ -0,0 +1,68 @@
+/* Copyright Ville Hiltunen 2018 <hiltunenvillej@gmail.com>
```

Comment:
I prefer a coding style where the first line in a file is empty and the second line gives the brief, one line explanation of the purpose of the file.